src/main/java/com/github/wasiqb/coteafs/selenium/core/base/driver/CustomScreenRecorder.java

```diff
@@ -0,0 +1,117 @@
+
+import static java.text.MessageFormat.format;
+import static org.monte.media.FormatKeys.EncodingKey;
+import static org.monte.media.FormatKeys.FrameRateKey;
+import static org.monte.media.FormatKeys.KeyFrameIntervalKey;
+import static org.monte.media.FormatKeys.MIME_AVI;
+import static org.monte.media.FormatKeys.MediaTypeKey;
+import static org.monte.media.FormatKeys.MimeTypeKey;
+import static org.monte.media.Registry.getInstance;
+import static org.monte.media.VideoFormatKeys.CompressorNameKey;
+import static org.monte.media.VideoFormatKeys.DepthKey;
+import static org.monte.media.VideoFormatKeys.ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE;
+import static org.monte.media.VideoFormatKeys.QualityKey;
+
+import java.awt.AWTException;
+import java.awt.Dimension;
+import java.awt.GraphicsConfiguration;
+import java.awt.GraphicsEnvironment;
+import java.awt.Rectangle;
+import java.awt.Toolkit;
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import com.github.wasiqb.coteafs.selenium.config.ConfigUtil;
+import com.github.wasiqb.coteafs.selenium.config.RecorderSetting;
+
+import org.monte.media.Format;
+import org.monte.media.FormatKeys.MediaType;
+import org.monte.media.math.Rational;
+import org.monte.screenrecorder.ScreenRecorder;
+
+/**
+ * @author Wasiq Bhamla
+ * @since 27-Oct-2019
+ */
+class CustomScreenRecorder extends ScreenRecorder {
+    private static final RecorderSetting RECORDER_SETTING = ConfigUtil.appSetting ()
+        .getPlayback ()
+        .getRecording ();
+    private static ScreenRecorder        screenRecorder;
+
+    static void startRecording () throws Exception {
+        if (checkIfEnabled ()) {
+            final File file = new File (RECORDER_SETTING.getPath ());
+
+            final Dimension screenSize = Toolkit.getDefaultToolkit ()
+                .getScreenSize ();
+            final int width = screenSize.width;
+            final int height = screenSize.height;
+
+            final Rectangle captureSize = new Rectangle (0, 0, width, height);
+
+            final GraphicsConfiguration gc = GraphicsEnvironment.getLocalGraphicsEnvironment ()
+                .getDefaultScreenDevice ()
+                .getDefaultConfiguration ();
+
+            screenRecorder = new CustomScreenRecorder (gc, captureSize,
+                new Format (MediaTypeKey, MediaType.FILE, MimeTypeKey, MIME_AVI),
+                new Format (MediaTypeKey, MediaType.VIDEO, EncodingKey, ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE,
+                    CompressorNameKey, ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE, DepthKey, 24, FrameRateKey,
+                    Rational.valueOf (15), QualityKey, 1.0f, KeyFrameIntervalKey, 15 * 60),
+                new Format (MediaTypeKey, MediaType.VIDEO, EncodingKey, "black", FrameRateKey, Rational.valueOf (30)),
+                null, file);
+
+            screenRecorder.start ();
+        }
+    }
+
+    static void stopRecording () throws Exception {
```

Comment:
Appropriate logs should be written in console/file when video recording is stopped.